analysis/alignment/feature_alignment.py

```diff
@@ -628,36 +628,39 @@ def handle_args():
 
     import ast
     parser = argparse.ArgumentParser(description = usage )
```

Comment:
hmm it may be more complicated than this as there are many ways to run the alignment, using iRT only (is cython lowss installed or not) or using decoys/not using decoys (when using `target_fdr`) 

I will think about it and maybe make a 2nd pull request for it